pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -5,43 +5,59 @@ title: Upgrade Kommander
 menuWeight: 20
 excerpt: Steps to upgrade Kommander via CLI
 beta: false
-enterprise: true
 ---
  
-This section describes how to upgrade your Kommander Management cluster and all Platform Applications to their latest versions. To prevent compatibility issues, you must first upgrade Kommander on your Management Cluster before upgrading to DKP.
+This section describes how to upgrade your Kommander Management cluster and all Platform Applications to their latest versions in networked, air-gapped and on-prem environments. To prevent compatibility issues, you must first upgrade Kommander on your Management Cluster before upgrading to DKP.
 
 <p class="message--note"><strong>NOTE: </strong>It is important you upgrade Kommander BEFORE upgrading the Kubernetes version (or Konvoy version for Managed Konvoy clusters) in attached clusters, due to the previous versions' incompatibility with 1.22.</p>
 
 ## Prerequisites
 
-- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- [Download][download_binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.
+- For air-gapped environments only: Download the [images bundle][images_bundle], [charts bundle][charts_bundle] and [Catalog Application charts bundle][cat_apps_bundle].  
+- For air-gapped environments only: [Load the Docker images into your Docker registry][load_images]
```

Comment:
we'll need a step for downloading the latest kommander application application definitions (step 3 [here](https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/#install-on-konvoy))

Reply:
@cbuto , I added the info on downloading Kommander application definitions and charts bundles, thank you for the info! Could you please check that lines 22 to 32 are correct? In that order and for air-gapped only?

Reply:
looks good to me! yep that would only be required for air-gapped install/upgrades